custom/icds_reports/reports/__init__.py

```diff
@@ -104,6 +104,8 @@ def report_context(self):
             }
 
     def get_report_context(self, data_provider):
+        if not self.filter_set:
```

Comment:
when would this happen?

Reply:
When the report page is loaded (without filters set).